dre-service/src/main/java/io/cdap/re/RulesDB.java

```diff
@@ -51,10 +52,13 @@
  * This class {@link RulesDB} manages all the connections defined. It manages the lifecycle of Rules and Rulebooks
  * including all CRUD operations.
  */
-public final class RulesDB {
+final class RulesDB {
 
   private static final Logger LOG = LoggerFactory.getLogger(RulesDB.class);
 
+  private static final Type LIST_STRING_TYPE = new TypeToken<List<String>>() {
+  }.getType();
```

Comment:
nit, put this on the previous line: `() { }.getType()`

Reply:
I just did, but IntelliJ keeps pushing to a new line if I format the file. Any ideas on how to change the formatting settings to avoid this new line?

Reply:
it's something in your intellij settings

Reply:
Fixed it.